src/components/nav/SideNav.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled';
+
+const SideNav = () => {
+  return (
+    <SideNavWrapper>
+      <ContentWrapper>
+        <div>menu</div>
+        <div>menu</div>
+        <div>menu</div>
+        <div>menu</div>
+      </ContentWrapper>
+    </SideNavWrapper>
+  );
+};
+
+const SideNavWrapper = styled.aside`
+  background-color: lightblue;
```

Comment:
아마 지금은 먼저 레이아웃 잡느라 배경색을 임의로 주신 것 같아요.
하지만 나중에 리팩토링을 줄이기 위해 위에 호진님 말처럼 theme을 적용하는게 어떨까요?
```suggestion
  color: ${({ theme }) => theme.colors.BACKGROUND};
```

Reply:
요거는 배경색 자체가 사라질 건데 theme으로 적용하면 리팩토링이 줄어드는 효과가 있나요?